perception/multi_object_tracker/include/multi_object_tracker/tracker/model/normal_vehicle_tracker.hpp

```diff
@@ -28,6 +28,7 @@ class NormalVehicleTracker : public Tracker
 private:
   autoware_auto_perception_msgs::msg::DetectedObject object_;
   rclcpp::Logger logger_;
+  int last_nearest_corner_index_ = -1;
```

Comment:
can you use std::optional?

Reply:
Fixed in dbbbf3d8a18a39a5f6f78931e2c82ce6ff9fc875.
I removed this initialization because `last_nearest_corner_index_` is set in constructor. (see [here](https://github.com/YoshiRi/autoware.universe/blob/feature/MOT_cooperate_shape_pose_estimation/perception/multi_object_tracker/src/tracker/model/big_vehicle_tracker.cpp#L143))